presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -291,6 +329,15 @@ private void testUnsupportedDataType(String databaseDataType)
                 Function.identity());
     }
 
+    public static DataType<String> jsonDataType()
+    {
+        return DataType.dataType(
+                "json",
+                JSON,
+                value -> format("JSON '%s'", value),
```

Comment:
This doesn't work if the value has single quotes, but that's probably fine for this test

Reply:
```
value -> {
    checkArgument(!value.contains("'"));
    return format("JSON '%s'", value);
}
```